src/trufont/objects/application.py

```diff
@@ -108,24 +109,47 @@ def newFont(self) -> Font:
         FontWindow(None, font).Show()
         return font
 
-    def openFont(self, path=None) -> Optional[Font]:
+    def openFont(self, path: Optional[str] = None) -> Font:
```

Comment:
The return type is Optional[Font] if you look at the method body

Reply:
Yes, caught that after mypy gave me a weird look.